frontend/src/static/js/components/webstatus-overview-table.ts

```diff
@@ -113,6 +117,54 @@ export class WebstatusOverviewTable extends LitElement {
     ];
   }
 
+  findFeaturesFromAtom(
+    searchKey: string,
+    SearchValue: string,
+  ): components['schemas']['Feature'][] {
+    if (!this.taskTracker.data?.data) {
+      return [];
+    }
+
+    const features: components['schemas']['Feature'][] = [];
+    for (const feature of this.taskTracker.data.data) {
+      if (searchKey === 'id' && feature?.feature_id === SearchValue) {
+        features.push(feature);
+        break;
+      } else if (
+        searchKey === 'name' &&
+        (feature?.feature_id.includes(SearchValue) ||
+          feature?.name.includes(SearchValue))
+      ) {
+        features.push(feature);
+      }
+    }
+    return features;
+  }
+
+  sortDataOrder(): components['schemas']['Feature'][] | undefined {
```

Comment:
```suggestion
  reorderByQueryTerms(): components['schemas']['Feature'][] | undefined {
```
Maybe rename this something specific to the QueryTerms. `reorderByQueryTerms` or `sortByQueryTerms`